packages/tools/src/utilities/math/polyline/combinePolyline.ts

```diff
@@ -397,4 +475,756 @@ function subtractPolylines(
   return subtractedPolylines;
 }
 
-export { mergePolylines, subtractPolylines };
+function subtractPolylines(
```

Comment:
can you break this file into multiple files, it is huuuuuuge, one for each i would say